modules/upgrade/nav-upgrade-guide.adoc

```diff
@@ -25,7 +25,8 @@ endif::[]
 ** xref:proxy-intro.adoc[Upgrade the Proxy]
 ifeval::[{uyuni-content} == true]
 // *** xref:proxy-autoyast-uyuni.adoc[Proxy - Upgrade with AutoYaST]
-*** xref:proxy-uyuni.adoc[Proxy - Upgrade Procedure]
+*** xref:proxy-uyuni.adoc[Proxy - Upgrade Procedure from 2021.05]
+*** xref:proxy-minor-uyuni.adoc[Proxy - Upgrade Procedure from 2021.06]
```

Comment:
Why do we want to mention versions here? Why can't we do this as for SUSE Manager with generic titles :-?

In fact that's what you use later at `modules/upgrade/pages/proxy-intro.adoc`